auditlog/models/rule.py

```diff
@@ -518,10 +529,14 @@ def create_logs(
             diff = DictDiffer(
                 new_values.get(res_id, EMPTY_DICT), old_values.get(res_id, EMPTY_DICT)
             )
+            origin_model = self.env[log.model_model].browse(log.res_id)
```

Comment:
- Maybe instead of `log.res_id` just use `res_id`
- Instead of `self.env[log.model_model]` you can just use `model_model` like defined above
- If you move this code up you can change `name = model_model.browse(res_id).name_get()` to `name = origin_model.name_get()` as well